src/visGeometry/GeometryStore.ts

```diff
@@ -43,6 +48,31 @@ class GeometryStore {
     private _cachedAssets: Map<string, string>;
     private _registry: Registry;
     public mlogger: ILogger;
+    public static sphereGeometry: SphereBufferGeometry =
+        new SphereBufferGeometry(1, 32, 32);
+    public static cubeGeometry: BoxBufferGeometry = new BoxBufferGeometry(
+        1,
+        1,
+        1
+    );
+
+    private static loadPrimitive = (
+        displayType: GeometryDisplayType,
+        url?: string
+    ) => {
+        if (!url) {
+            // if there isn't an url to load, even if they selected PDB or OBJ
+            // we have to default to a sphere. May change depending on how we handle the gizmo
```

Comment:
I think this is really trying to say that some displaytypes require a url to work.

if there isn't a url AND they selected PDB or OBJ, then we have to default to a sphere.  Otherwise we can use the specified type that doesn't require a url.
Is that correct?

Reply:
`if displaytyperequiresurl(displayType) && !url) { use sphere } ...` ?

Reply:
yeah that's what I was trying to say in the comment